src/main/java/Task.java

```diff
@@ -0,0 +1,33 @@
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+    public Task(String description) {
+        this.description = description;
+        isDone = false;
+    }
+    public String getStatusIcon() {
+        return (isDone ? "X" : " "); // mark done
```

Comment:
Coding Standard: Comments should be indented relative to their position in the code.

Consider putting it in the previous line on the same indentation as the return statement.

Additionally consider phrasing it to "write to the reader" as per the code quality guiideline for comments, which will be covered later in the module.